cmd/operator/main.go

```diff
@@ -35,9 +35,9 @@ import (
 	"syscall"
 
 	"github.com/Sirupsen/logrus"
-	"github.com/heptiolabs/healthcheck"
-	"github.com/prometheus/client_golang/prometheus"
-	"github.com/prometheus/client_golang/prometheus/promhttp"
+//	"github.com/heptiolabs/healthcheck"
```

Comment:
Leftover comments in this file?

Reply:
No change is required in main,go, I have commented out few lines because of compilation error, I will undo this in the next commit.